src/test/java/duke/test/DukeTest.java

```diff
@@ -0,0 +1,15 @@
+package duke.test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Perhaps be more consistent with the ordering of imports, with static imports only above or below non-static imports.